docker/images/brownie/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.9
+FROM --platform linux/amd64 python:3.9

```

Comment:
I'm getting the following error when using this change:
"failed to solve with frontend dockerfile.v0: failed to create LLB definition: dockerfile parse error line 1: FROM requires either one or three arguments"